java/core/src/test/java/co/worklytics/psoxy/rules/asana/AsanaTests.java

```diff
@@ -0,0 +1,178 @@
+package co.worklytics.psoxy.rules.asana;
+
+import co.worklytics.psoxy.rules.JavaRulesTestBaseCase;
+import co.worklytics.psoxy.rules.RuleSet;
+import lombok.Getter;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Stream;
+
+public class AsanaTests extends JavaRulesTestBaseCase {
+
+    @Getter
+    final RuleSet rulesUnderTest = PrebuiltSanitizerRules.ASANA;
+
+    @Getter
+    final String exampleDirectoryPath = "api-response-examples/asana";
+
+    @Getter
+    final String defaultScopeId = "asana";
+
+    @Getter
+    final String yamlSerializationFilepath = "asana/asana";
+
+    @Disabled // not reliable; seems to have different value via IntelliJ/AWS deployment and my
+    // laptop's maven, which doesn't make any sense, given that binary deployed to AWS was built via
+    // maven on my laptop - so something about Surefire/Junit runner used by maven??
+    @Test
+    void sha() {
+        this.assertSha("7869e465607b7a00b4bd75a832a9ed1f811ce7f2");
```

Comment:
This SHA1 is from other set of rules... still not working?

Reply:
it's platform-dependent, it appears